lib/aws_loader.rb

```diff
@@ -1,8 +1,8 @@
 require 'yaml'
-require_relative '../config/q_it'
+require_relative '../config/application'
 module AwsLoader
   
-  AWS_PATH = File.join(QIt.root, 'config/aws.yml')
+  AWS_PATH = File.join(Application.root, 'config/aws.yml')
   AWS = YAML.load(File.read(AWS_PATH))["aws"] if File.exists?(AWS_PATH)
```

Comment:
Are you using `AWS` anywhere - if not you can remove this line.

Reply:
Please check the above comment - and address it.

Reply:
Using AWS constant in the client to access the key values.

Reply:
Ok.. then keep it in..